eng/Signing.props

```diff
@@ -1,4 +1,10 @@
 <Project>
+  <ItemGroup>
+    <ItemsToSign Include="$(ArtifactsBinDir)\Mono.Cecil\$(Configuration)\netstandard2.0\Mono.Cecil.dll" />
+    <ItemsToSign Include="$(ArtifactsBinDir)\Mono.Cecil.Rocks\$(Configuration)\netstandard2.0\Mono.Cecil.Rocks.dll" />
+    <ItemsToSign Include="$(ArtifactsBinDir)\Mono.Cecil.Mdb\$(Configuration)\netstandard2.0\Mono.Cecil.Mdb.dll" />
+    <ItemsToSign Include="$(ArtifactsBinDir)\Mono.Cecil.Pdb\$(Configuration)\netstandard2.0\Mono.Cecil.Pdb.dll" />
```

Comment:
I don't have a clear idea of what caused it. After moving the packaging to a different project arcade started to complain about it ".packages\microsoft.dotnet.arcade.sdk\8.0.0-beta.23103.1\tools\Sign.proj(37,5): error : List of files to sign is empty. Make sure that ItemsToSign is configured correctly." so I just added the list. I know usually has a default list of items, but the packaging project is using a special SDK to be a project that doesn't compile so that might be the problem.